packages/core/src/controllers/TransactionsController.ts

```diff
@@ -12,6 +13,7 @@ type TransactionByYearMap = Record<number, TransactionByMonthMap>
 export interface TransactionsControllerState {
   transactions: Transaction[]
   transactionsByYear: TransactionByYearMap
+  lastNetworkInView: string | undefined
```

Comment:
Should this be that stricter `${string}:${string}` type by any chance? We might also have a type like CaipChain around that you could reuse for these.